src/components/input/input.scss

```diff
@@ -33,6 +39,15 @@ $md-input-underline-disabled-background-image: linear-gradient(to right,
   }
 }
 
+/**
+ * Applies a floating placeholder above the input before chrome's and safari's auto-fill commits.
+ * Firefox has selector issues for :-webkit-autofill, thats why this mixin is duplicated and is used seperately below 
+ */
+%md-input-placeholder-floating-autofill {
+  visibility: visible;
+  padding-bottom: 5px;
+  transform: translateY(-100%) scale(0.75);
+}
```

Comment:
These definitions are scss **placeholders**, not mixins, which is the problem. See 
http://thesassway.com/intermediate/understanding-placeholder-selectors

If you change to a single _mixin_, it should be fine.